src/Surfnet/ServiceProviderDashboard/Application/Command/Entity/SaveOidcEntityCommand.php

```diff
@@ -482,17 +482,25 @@ public function setEnvironment($environment)
     /**
      * @return string
      */
-    public function getClientId()
+    public function getEntityId()
+    {
+        return $this->entityId;
+    }
+
+    /**
+     * @param string $entityId
+     */
+    public function setEntityId($entityId)
     {
-        return $this->clientId;
+        $this->entityId = $entityId;
     }
 
     /**
-     * @param string $clientId
+     * @return string
      */
-    public function setClientId($clientId)
+    public function getClientId()
```

Comment:
Is this getter actually used? I could not find any usages.

Reply:
`\Surfnet\ServiceProviderDashboard\Infrastructure\DashboardBundle\Form\Entity\OidcEntityType`

Reply:
The clientId is only for the presentation of the clientId.

Reply:
Ahh, yes